app/views/thredded/posts/_content_field.html.erb

```diff
@@ -1,4 +1,6 @@
 <li>
   <%= form.label :content, content_label %>
+  <%= render 'thredded/posts/before_content_field', form: form %>
   <%= form.text_area :content, { rows: 5, required: true }  %>
+  <%= render 'thredded/posts/after_content_field', form: form %>
```

Comment:
@glebm thinking about this a little more. What do you think about `[before|after]_textarea` for these partials instead? Because `[before|after]_content` could be misconstrued as being, literally, before and after a post's contents.